### ISSUE TITLE
accuracy exceed 1.8000

### ISSUE THREAD

**Issue by AloshkaD** [jocicmarko/ultrasound-nerve-segmentation#46]
@jocicmarko Thank you for sharing your implementation.

I'm using the model for training on 10000+ rgb images and the dice-coef exceeds 1.8000 (loss is obviously negative of that number). Appreciate your thoughts on that.

**Comment by luckustus** [comment]
Same issue, my dice coefficient seems to exceed 1 from time to time. Seems to be related to the batch size

**Comment by jocicmarko** [comment]
Before I investigate this further, can you provide me some details? 

- Do you still use sigmoid as an activation function?
- Did you make any changes to outputs? I.e. more output classes instead of binary
- Did you make sure you scaled your ground truth values to [0,1]

**Comment by luckustus** [comment]
Yes, sigmoid. Output is binary and the ground truth values are 0 and 1.
THe only thing that I am doing differently is feeding the data. I am using a data generator that produces arrays of shape (batch_size,height, width,channels) and I am using TF backend:

model.fit_generator(DataGenerator(img_list,,batch_size) ,steps_per_epoch = 570,epochs = 50)

**Comment by akirakitamura19** [comment]
Did anyone solve this problem?
When I apply this model for different dataset, same issue happened. 
In my case, I don't use data generator for implementing.

**Comment by luckustus** [comment]
Most likely something is wrong with your input that you provide. I don't remember exactly what the issue was but I fixed it by fixing the input.

**Comment by akirakitamura19** [comment]
@luckustus 
I found easy careless mistakes which made mask input [0-255]. Now it works well. Thanks.

**Comment by AloshkaD** [comment]
@akirakitamura19 is that a normalization layer that you are talking about?

**Comment by akirakitamura19** [comment]
@AloshkaD Yes, it is. Just make sure that values of train and train_make are [0 - 1].
